### PR TITLE
Add updatetree Helm Chart to mark old shards as frozen

### PR DESCRIPTION
**Pull request by priyawadhwa**
This can be used to mark old shards as FROZEN, or as ACTIVE if we ever need to rollback sharding.

Signed-off-by: Priya Wadhwa <priya@chainguard.dev>

 **Description of the change**

 **Existing or Associated Issue(s)**

 **Additional Information**

 

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [x] Variables are documented in the `values.yaml` and added to the README.md. THe [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content
- [x] JSON Schema generated
- [x] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command

### CODE REVIEW

File: charts/updatetree/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+| Name | Email | Url |
+| ---- | ------ | --- |
+| The Sigstore Authors |  |  |
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| args.treeID | string | `nil` |  |
+| args.treeState | string | `nil` |  |
```
Comment (k4leung4) [inline charts/updatetree/README.md:20]: maybe in the description, spell out what are valid treeState values?

Comment (priyawadhwa) [reply charts/updatetree/README.md]: good call, done

File: charts/updatetree/values.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+namespace:
+  name: trillian-system
+serviceAccount:
+  name: trillian-logserver
+  create: false
```
Comment (k4leung4) [inline charts/updatetree/values.yaml:5]: can you add `annotations`, so it would show up in the README.md and schema.

thanks

Comment (priyawadhwa) [reply charts/updatetree/values.yaml:6]: done!

**Review by k4leung4 — APPROVED** [review]

**Pull request merged**